src/barChart.ts

```diff
@@ -130,6 +140,57 @@ module powerbi.extensibility.visual {
         };
     }
 
+    function getColumnColorByIndex(
+        category: DataViewCategoryColumn,
+        index: number,
+        colorPalette: ISandboxExtendedColorPalette,
+    ): string {
+        if (colorPalette.isHighContrast) {
+            return colorPalette.background.value;
+        }
+
+        const defaultColor: Fill = {
+            solid: {
+                color: colorPalette.getColor(`${category.values[index]}`).value,
+            }
+        };
+
+        return getCategoricalObjectValue<Fill>(
+            category,
+            index,
+            'colorSelector',
+            'fill',
+            defaultColor
+        ).solid.color;
+    }
+
+    function getColumnStrokeColor(colorPalette: ISandboxExtendedColorPalette): string {
+        return colorPalette.isHighContrast
+            ? colorPalette.foreground.value
+            : null;
+    }
+
+    function getAxisTextFillColor(
```

Comment:
I can't spot the bug here, but when testing the visual, axis text is always black, it should be foreground color.

Reply:
done. Issue was related to applying `color` instead of `fill`